pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1018,46 +1192,54 @@ def __init__(
         ):
             raise ValueError("The work wires must be different from the control and target wires")
 
-        self._target_wire = wires[0]
-        self._work_wires = work_wires
-        self._control_wires = control_wires
-
-        wires = control_wires + wires
-
         if not control_values:
             control_values = "1" * len(control_wires)
 
-        control_int = self._parse_control_values(control_wires, control_values)
-        self.control_values = control_values
+        # we do not store this in the hyperparameters,
+        # because it influences the implementation of this op,
+        # not the transformation itself
+        self.work_wires = work_wires
```

Comment:
not 100% sure I follow this comment?

Reply:
I think Christina adds this to the hyperparams. But it is not needed to represent the matrix, so in my books it is not a hyperparameter :) But maybe I have to widen my definition a bit, since it steers the decomposition...